packages/recommend-core/package.json

```
@@ -28,7 +28,7 @@
     "prepare": "yarn build:esm && yarn build:types"
   },
   "peerDependencies": {
-    "@algolia/recommend": "4.10.2"
+    "@algolia/recommend": "4.10.5"
```

```suggestion
    "@algolia/recommend": "^4.10.5"
```

should this not be a caret regardless of this PR?

---

Good catch! This might have been missed in the past :D